1-exercises/J-string-substring/exercise2.js

```diff
@@ -14,10 +14,10 @@ let names = [
   "Arron Graham",
 ];
 
-names[0] = names[0].substring();
-names[1] = names[1].substring();
-names[2] = names[2].substring();
-names[3] = names[3].substring();
+names[0] = names[0].substring(0, 6);
+names[1] = names[1].substring(0, 7);
+names[2] = names[2].substring(0, 4);
+names[3] = names[3].substring(0, 4);
```

Comment:
We want to calculate rather than hardcoding the position to substring. Consider using **indexOf** or **split**